packages/front-end/components/Radix/PremiumCallout.tsx

```diff
@@ -18,6 +18,7 @@ export type Props = {
   renderWhenDismissed?: (undismiss: () => void) => React.ReactElement;
   children: React.ReactNode;
   docSection?: DocSection;
+  cta?: React.ReactElement;
```

Comment:
The `docSection` prop is really just a shortcut for doing `cta={<DocLink docSection="...">View Docs</DocLink>}`.

I would get rid of the `docSection` prop and also change the logic here slightly.  I think the `Upgrade Now` and `Talk to Sales` CTAs should take precedence.  So only render this `cta` prop when the user already has the feature.